examples/axes_grid1/simple_axes_divider1.py

```diff
@@ -2,17 +2,25 @@
 =====================
 Simple Axes Divider 1
 =====================
-
 """
 
 from mpl_toolkits.axes_grid1 import Size, Divider
 import matplotlib.pyplot as plt
 
 
+def setup(ax, text):
```

Comment:
```suggestion
def axes_label(ax, text):
```

Does a bit more, than the name says, but you have the docstring. `setup()` is too generic.

Reply:
sure